### DISCUSSION TITLE
What's your profile

### DISCUSSION THREAD

**Discussion by DominiqueMakowski** [DominiqueMakowski/teaching#5] [General]
What's your GH page?

**Comment by Pascal-Kueng** [comment]
https://github.com/Pascal-Kueng

**Reply by DominiqueMakowski** [reply]
Haha don't go too fast or you will get bored during the course :p

**Comment by DominiqueMakowski** [comment]
https://github.com/DominiqueMakowski

**Comment by Cymelo92** [comment]
Hello!

**Reply by Cymelo92** [reply]
https://github.com/Cymelo92/Cymelo92

**Reply by Cymelo92** [reply]
```python

def isOdd(x):
    if isinstance(x, str):
        print("please enter a float or an integer.")
    else:
        if (x % 2) == 0:
            print(f"{x} is even")
        else:
            print(f"{x} is odd")
```

**Comment by MartynaPlomecka** [comment]
https://github.com/MartynaPlomecka/MartynaPlomecka

**Comment by nriese** [comment]
https://github.com/nriese

**Comment by zitamayer** [comment]
https://github.com/zitamayer

**Comment by LenaSchn** [comment]
https://github.com/LenaSchn

**Comment by NimsajB** [comment]
https://github.com/NimsajB/NimsajB

**Comment by SvennoNito** [comment]
https://github.com/SvennoNito/Svennonito

**Comment by GeorgiaSou** [comment]
https://github.com/GeorgiaSou/GeorgiaSou

**Comment by Becca495** [comment]
https://github.com/Becca495/Rebecca_PythonCourseDom

**Reply by Becca495** [reply]
https://github.com/Becca495

**Comment by rejoha** [comment]
https://github.com/rejoha

**Comment by AnaRodMa** [comment]
https://github.com/AnaRodMa/AnaRodMa

**Comment by floramouj** [comment]
https://github.com/floramouj/floramouj

**Comment by NimsajB** [comment]
the pip install command worked with pip3 install black. Maybe it has to do with the version

**Reply by nriese** [reply]
This didn't work for me either, unfortunately. So far I could not resolve the issue for me - does anyone else also still has the problem? (maybe it's a windows-problem?)

**Reply by DominiqueMakowski** [reply]
@nriese what's the message error when you try running pip?

**Reply by DominiqueMakowski** [reply]
perhaps this  can help https://stackoverflow.com/a/71595720

**Reply by nriese** [reply]
This was the error message:

_PS C:\Users\Nathalie Rieser> pip install black
pip : The term 'pip' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of 
the name, or if a path was included, verify that the path is correct and try again.
At line:1 char:1
+ pip install black
+ ~~~
    + CategoryInfo          : ObjectNotFound: (pip:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException_

**But I now fixed the issue with #8:** https://stackoverflow.com/questions/36835341/pip-is-not-recognized
(Adding Python to the PATH in a different way didn't work)

**Reply by DominiqueMakowski** [reply]
Tagging @SvennoNito who also had issues if I remember
